packages/x-date-pickers/src/MobileDatePicker/MobileDatePicker.test.tsx

```diff
@@ -206,7 +206,7 @@ describe('<MobileDatePicker />', () => {
       />,
     );
     const start = adapterToUse.date();
-    fireEvent.click(screen.getByRole('textbox'));
+    fireEvent.mouseDown(screen.getByRole('textbox'));
```

Comment:
For context, in the Select component, we have seen a few developer struggle with figuring out how to open the popup once we moved from onClick to onMouseDown. I don't remember how we fixed this in the Select and in the Autocomplete, but anyway, it's for later, once developers open issues about it.

Reply:
Using a more [abstract API](https://testing-library.com/docs/user-event/convenience#click) (higher-level) for writing tests might help. 🤔

Reply:
Related to https://github.com/mui/mui-x/pull/5920

I believe our higher level API is `userEvent.mousePress`